test/common/router/config_impl_test.cc

```diff
@@ -6088,11 +6085,10 @@ name: foo
           seconds: 123
 )EOF";
 
-    checkNoPerFilterConfig(yaml);
-  }
+  checkNoPerFilterConfig(yaml);
 }
 
-TEST_F(PerFilterConfigsTest, RouteLocalConfig) {
+TEST_F(PerFilterConfigsTest, DEPRECATED_FEATURE_TEST(RouteLocalConfig)) {
```

Comment:
Why are only some of the tests upgraded, while others are left behind. Do we want these tests to be deleted in v3?

Reply:
All of test methods, except this one (RouteLocalConfig) tested both Any and Struct based configurations in the same test method. I have split these test methods into two testing Any and Struct separately and deprecating the Struct tests. The RouteLocalConfig was already split into separate methods.
The TypedConfigFilterError test above tests error condition when both Struct and Any are present in the config and thus deprecated as well, as the Struct field is removed in v3.